StortSpelProjekt/Sandbox/src/main.cpp

```diff
@@ -651,10 +655,21 @@ Scene* TimScene(SceneManager* sm)
     tc = entity->AddComponent<component::TransformComponent>();
     ccc = entity->AddComponent<component::CubeCollisionComponent>(0.0,35.0,0.0,35.0);
 
+    /*
+    HeightMapInfo inf;
```

Comment:
Will this comment be used?

Reply:
Yes, this branch is intended to be used as a base for the collision detection of heightmaps later. The code commented out will be reused and reedited.